File: leetcode/2951-find-the-peaks/findPeaks.go
package leetcode

func findPeaks(mountain []int) []int {
	var peaks = []int{}

	for index := 1; index < len(mountain)-1; index++ {
		var item = mountain[index]
		if item > mountain[index-1] && item > mountain[index+1] {
			peaks = append(peaks, index)
		}
	}

	return peaks
}

File: leetcode/2951-find-the-peaks/findPeaks.ts
function findPeaks(mountain: number[]): number[] {
  const peaks: number[] = [];

  for (let index = 1; index < mountain.length - 1; index++) {
    const item = mountain[index];

    if (item > mountain[index - 1] && item > mountain[index + 1]) {
      peaks.push(index);
    }
  }

  return peaks;
}

export default findPeaks;
